### PR TITLE
Otto-App UI Enhancements Pt1.

### PR DESCRIPTION
**Pull request by darkbrewery**
1. Add level badges to attributes in /my-ottos
image

2. Add Otter Level in /my-ottos/xxxx
image
-Icon provided by Appppo

3. Add icons to attributes in /my-ottos/xxxx, centered text and added rounded border section
image
- Icons provided by Appppo

4. Change "View the previous adventure journal here >" in /adventure (resting/ready )to a button with icon
-Includes adding translation for the word "Results" into Chinese simple/trad.

5. Update general layout of otter cards in /adventure to be more concise and pretty. 
image
 -Ready/Resting/Ready/Ongoing colour badges are now always in the same place, easier to reference in the "all" tab
 -Added text that was missing for finished adventures so the layout stays consistent "x has finished exploring x!" (with translation)
-Card background has a faint image depicting the location that is active, help visualize distribution
-Level and location badges have had their spacing fixed up and made more readable
-Otto image increased in size, no more squinting!
-Buttons and timers are all now the same size, more consistent and organised.
-Tested on mobile, elements realign in a pleasing way on low resolution devices.

6. Started optimizing overly large images in the app using lossless compression. (More in Pt2)
image
-The homepage city image(as beautiful as it is) is an unnecessary 8k pixels wide and over 30mb, have added a new version that's only 2mb (95% smaller).
-Apx 70+ additional images used in the app were uncompressed and slowing down pageload time significantly. Average bandwidth savings of 35%.
-This generally isn't an issue for desktop browsers that cache the files locally, but it's a very real issue for users with mobile wallet browsers that do not cache any files.

### CODE REVIEW

File: src/views/otto/OttoPage/index.tsx [hunk 1]
```diff
@@ -445,28 +477,40 @@ export default function OttoPage() {
...
-                </Button>
-              </StyledAttributePoints>
-            )}
+            <StyledStat>
+              <StyledAttrs>
+                {otto?.displayAttrs.map(({ trait_type, value }, index) => (
+                  <StyledAttr key={index}>
+                    <StyledAttrLabel>
+                      {AttrIconsMap[trait_type] && (
+                        <StyledAttrIcon icon={AttrIconsMap[trait_type]} />
```
Comment (pyc92) [inline src/views/otto/OttoPage/index.tsx:486]: missing `key` prop

**Comment by pyc92** [comment]
@darkbrewery we compressed all images via the pre-commit hook, the hook seems not working correctly now.
here the compression script: https://github.com/OtterClam/otto-app/blob/main/scripts/imagemin.mjs

could you also fix the hook and update the script?

**Comment by darkbrewery** [comment]
> 

Done!

**Comment by darkbrewery** [comment]
imagemin will only run on new files comitted

**Comment by darkbrewery** [comment]
Pretty sure I've got it now

**Comment by pyc92** [comment]
cool!

**Pull request merged**